src/components/button/button.spec.tsx

```diff
@@ -102,6 +133,57 @@ describe("Button", () => {
     });
   });
 
+  describe( "Button Minor Padding",() => {
+      it.each(isMinorSizesPadding)("when size is %s the padding is %s", (size, padding) => {
+          const wrapper = mount(
+            <Button isMinor size={size}>Foo</Button>
+          )
+          assertStyleMatch({padding}, wrapper)    
+      });
+  });
+
+  describe( "Button Minor Colors ",() => {
```

Comment:
```suggestion
  describe("Button Minor Colors ",() => {
```

Reply:
If you don't make the change describe here https://github.com/Sage/carbon/pull/5553#discussion_r1015293812